test-browser/websocket.ts

```diff
@@ -0,0 +1,670 @@
+import { assert, expect, test, beforeEach, vi } from "vitest";
+import { AMQPWebSocketClient } from '../src/amqp-websocket-client.js';
+import { AMQPMessage } from '../src/amqp-message.js';
+import type { AMQPError } from "../src/amqp-error.js";
+
+beforeEach(() => {
+  expect.hasAssertions()
+})
+
+test('can parse the url correctly', () => {
+  const username = 'user_name'
+  const password = 'passwd'
+  const hostname = '127.0.0.1'
+  const port = 15670
+  const vhost = 'my_host'
+  const name = 'test'
+  const client = new AMQPWebSocketClient({ url: `ws://${hostname}:${port}/ws/amqp`, username: username, password: password, vhost: vhost, name: name });
+  expect(client.username).toEqual(username);
+  expect(client.password).toEqual(password);
+  //expect(client.host).toEqual(hostname);
+  //expect(client.port).toEqual(port);
```

Comment:
I think we can just remove these commented lines

Reply:
Remvoed